1-js/02-first-steps/05-types/article.md

```diff
@@ -43,23 +43,23 @@ n = 12.345;
 - `NaN` بیانگر یک اشکال محاسباتی است. این مقدار نتیجه یک عملیات ریاضیاتی غلط یا تعریف‌نشده است. برای نمونه:
 
     ```js run
-    alert( "چیزی غیر از عدد" / 2 ); // NaN ،چنین تقسیمی اشتباه است
+    alert( "not a number" / 2 ); // NaN ،چنین تقسیمی اشتباه است
```

Comment:
This is a good suggestion for reading the syntax in a correct way.
It would be great to have the translation in paranthesis too, so the reader knows the meaning.

```suggestion
    alert( "not a number (چیزی غیر از عدد)" / 2 ); // NaN ،چنین تقسیمی اشتباه است
```